src/components/clouds/style.ts

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+import { colors } from "src/constants";
+
+export const Container = styled.div`
```

Comment:
We have default margin's and padding's for all block's. If it no like this, lets add default dor all div's in global styles. Remove this container if you use it only for remove margin and padding's